api/app/serializers/maker_partial_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# Provides a partial serialization of a maker model.
+class MakerPartialSerializer < ActiveModel::Serializer
+  meta(partial: false)
+
+  cache key: "maker", expires_in: 3.hours
```

Comment:
We're going to stop caching these responses for now, so go ahead and remove this. I removed it from all the other serializers. For the reason behind this, see the warning here:

https://github.com/rails-api/active_model_serializers/blob/master/docs/general/caching.md